### PR TITLE
document how to rotate release signing key

### PR DESCRIPTION
**Pull request by tillcoco**
This documents how to rotate the release signing key. In fact, it documents what I did to rotate it. There's probably room for improvement.

### CODE REVIEW

File: HOW-TO-RELEASE.md [hunk 1]
```diff
@@ -23,3 +23,30 @@ Wait for the [CI pipeline](../../actions) to release the new version. Publishing
...
+* for email address, use `info@commercetools.com`
+
+If the programmatic public key publishing fails for you (`gpg: keyserver send failed: Server indicated a failure`), you can publish the public key manually.
+Use the forms on https://keyserver.ubuntu.com/ and http://pgp.mit.edu:11371/.
+It's important to add the public key to both key servers!
+
+Finally, update the `PGP_PASSPHRASE` and `PGP_SECRET` secrets in the [repository settings](https://github.com/commercetools/sphere-scala-libs/settings/secrets/actions).
+
+_There is no need to save the key details anywhere else because it's only used for signing and will never be verified.
+If the credentials are lost, we can just generate a new key._
```
Comment (tillcoco) [inline HOW-TO-RELEASE.md:49]: Please confirm.

File: HOW-TO-RELEASE.md [hunk 2]
```diff
@@ -23,3 +23,30 @@ Wait for the [CI pipeline](../../actions) to release the new version. Publishing
...
+
+If you generated the key using `gpg --gen-key`, its output included an `[expires: YYYY-MM-DD]` string.
```
Comment (tillcoco) [inline HOW-TO-RELEASE.md:51]: I guess, we can also choose to generate a key w/o expiration date but I'm not sure if that has any drawbacks. Will the public key servers also keep it forever?

File: HOW-TO-RELEASE.md [hunk 3]
```diff
@@ -23,3 +23,30 @@ Wait for the [CI pipeline](../../actions) to release the new version. Publishing
...
+Please make sure to set a reminder to rotate the key before it expires.
```
Comment (tillcoco) [inline HOW-TO-RELEASE.md:52]: When discussing this, please keep in mind that this is a public repository. The discussion might better be done internally.

**Review by yanns — APPROVED** [review]
💯

File: HOW-TO-RELEASE.md (hunk 1 above)
Comment (adrobisch) [reply HOW-TO-RELEASE.md:49]: works for me

**Review by adrobisch — APPROVED** [review]

**Pull request merged**